src/utils/index.ts

```diff
@@ -150,7 +150,7 @@ export const isOGMessage = (message: UserMessage | FileMessage): boolean => !!(
     || message.ogMetaData?.defaultImage
   )
 );
-export const isTextMessage = (message: UserMessage | FileMessage): boolean => isUserMessage(message) && !isOGMessage(message);
```

Comment:
FYI, I removed the dependency between isTextMessage <-> check isOGMessage util function but handled this logic in the components instead.

Reply:
Nit/personal-taste (dont need to apply) 
Hmmm, on second thought, better to have handled here.. would be more testable / concise